src/lib/autocomplete/autocomplete.ts

```diff
@@ -153,9 +153,16 @@ export class MatAutocomplete extends _MatAutocompleteMixinBase implements AfterC
   @Input('class')
   set classList(value: string) {
     if (value && value.length) {
-      value.split(' ').forEach(className => this._classList[className.trim()] = true);
-      this._elementRef.nativeElement.className = '';
+      this._classList = value.split(' ').reduce((classList, className) => {
+        classList[className.trim()] = true;
+        return classList;
+      }, {} as {[key: string]: boolean});
+    } else {
+      this._classList = {};
     }
+
+    this._setVisibilityClasses(this._classList);
+    this._elementRef.nativeElement.className = '';
```

Comment:
I see it was like this before, but doesn't this wipe out `.mat-autocomplete`?

Reply:
It does, but that's the point. This is the element that is left behind in the view after it provides the `ng-template` for the overlay panel.